kafka-connect-aws-s3/src/it/scala/io/lenses/streamreactor/connect/aws/s3/SlowTest.scala

```diff
@@ -1,5 +1,5 @@
 package io.lenses.streamreactor.connect.aws.s3
+
 import org.scalatest.Tag
 
 object SlowTest extends Tag("io.lenses.tags.SlowTest")
```

Comment:
Did you mention that these tags can be deleted now?

Reply:
Missed that. Thanks. I will remove them.